velox/functions/lib/Re2Functions.cpp

```diff
@@ -817,6 +1070,87 @@ std::shared_ptr<VectorFunction> makeRe2Extract(
   }
 }
 
+std::vector<std::shared_ptr<exec::FunctionSignature>> re2ReplaceSignatures() {
+  return {
+
+      // regex_replace (str, pattern, replace)
+      exec::FunctionSignatureBuilder()
+          .returnType("varchar")
+          .argumentType("varchar")
+          .argumentType("varchar")
+          .argumentType("varchar")
```

Comment:
make varchar, integer, bigint constant string and use the constants here.

Reply:
Updated to leverage TypeKind enum